src/distPlus/distribution/Mixed.re

```diff
@@ -1,7 +1,7 @@
 open Distributions;
 
 type t = DistTypes.mixedShape;
-let make = (~continuous, ~discrete): t => {continuous, discrete};
+let make = (~continuous, ~discrete, integralSumCache, integralCache): t => {continuous, discrete, integralSumCache, integralCache};
```

Comment:
why aren't the caches ~named? They seem like the less important bits. Typically either all params are named, or the one key one is not named.